src/app/feature/feature.component.ts

```diff
@@ -21,7 +21,7 @@ interface CurrentImagesCache {
     encapsulation: ViewEncapsulation.None
 })
 export class FeatureComponent implements OnInit {
-    @ViewChild('slideshow', { static: false }) public slideshow: any;
+    @ViewChild('slideshow') public slideshow: any;
```

Comment:
Did you test that this still works? I think it had a reason I added the `static` param, but it was a while ago...

Reply:
yes, angular removed that automatically during the update